intel_realsense_payload_for_spotsim2real/spotsim2realintegration_files/img_publishers_.py

```diff
@@ -0,0 +1,499 @@
+# Copyright (c) Meta Platforms, Inc. and its affiliates.
```

Comment:
this is okay for now, but we can integrate this with the existing `img_publishers.py`

Reply:
@jimmytyyang  If we merge semantic place PR, it already has the integration in spot.py & img_publishers.py in it thus deleting these files.